flutter_cache_manager_firebase/pubspec.yaml

```
@@ -10,10 +10,10 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_cache_manager: ^1.0.0
-  firebase_storage: ^3.0.0
+  firebase_storage: ^4.0.0
```

If there are no breaking changes, why not support both version 3 and 4?
```suggestion
  firebase_storage: '>=3.0.0 <5.0.0'
```

---

Yes, you are correct, there's no breaking changes